Cargo.toml

```diff
@@ -87,6 +87,7 @@ members = [
     "validator_client",
     "validator_client/slashing_protection",
 ]
+resolver = "2"
```

Comment:
This resolver change was necessary to support only enabling the `background_threads` feature on Linux.